app/src/test/java/com/iitp/njack/iitp_connect/core/home/AuthRepositoryTest.java

```diff
@@ -0,0 +1,32 @@
+package com.iitp.njack.iitp_connect.core.home;
+
+
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.mockito.InjectMocks;
+import org.mockito.Mock;
+import org.mockito.junit.MockitoJUnitRunner;
+
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.equalTo;
+
+@RunWith(MockitoJUnitRunner.class)
+public class AuthRepositoryTest {
+
+    @InjectMocks
+    private AuthRepository authRepository;
+
+
+    @Mock
+    private FirebaseAuthLiveData firebaseAuthLiveData;
+
+
+    @Test
+    public void it_should_return_fireBase_live_data() {
```

Comment:
Don't use the snake case. It's Java. Please use camel casing. It should be something like `shouldReturnFirebaseLiveData`.